frontend/src/hooks/useAlert.tsx

```diff
@@ -45,9 +57,9 @@ function AlertProvider({ children }: AlertProviderProps) {
     queue: [],
   });
 
-  function addAlert(message: any, options: any) {
+  function addAlert(message: any, autoHideDuration: number | null = 4000) {
```

Comment:
I think the type is just `number` since it has a default

Reply:
Default value is for "if you don't pass the value". I have to pass `null` value for some cases. (for example, when downloading the adm322, we don't know how long it takes from backend, so passing `null` to make the alert keeps running without auto hide.